<div style="background: linear-gradient(135deg, #123b5d, #1f6f8b); color: white; padding: 28px; border-radius: 12px;">
  <h1 style="margin: 0;">GBU 2301: Business Statistics</h1>
  <h2 style="margin: 8px 0 18px 0; font-weight: 400;">Week 1: Course Introduction | Data and Statistics</h2>
  <p style="margin: 0; font-size: 1.05em;">Fall 2026 | Dr. Omer Cayirli | Assistant Professor of Finance</p>
</div>

Required reading: Anderson et al., Statistics for Business and Economics, 14th ed., Chapter 1

---

### Outline

* Intro
    * Course Content and Objectives
    * Course Materials
    * Lecturer
    * Rules and Grading
* Data and Statistics
    * Data and Data Sets
    * Types and Structures of Data
    * Data Sources and Statistical Studies
    * Descriptive Statistics and Statistical Inference
    * Analytics, Computing, and Ethical Practice

---

### Course Content

* Data and Statistics
* Descriptive Statistics
* Probability
* Sampling and Sampling Distributions
* Interval Estimation
* Hypothesis Testing
* Inference
* Simple Linear Regression

---

### Objectives

* Acquire the required knowledge of the quantitative fundamentals of statistics.
* Perform data analysis using modern statistical software.
* Evaluate data analyses and studies critically.
* Translate business problems into statistical questions.
* Select appropriate statistical methods.
* Interpret statistical results in their business context.

---

### Course Materials

#### Book

* Statistics for Business and Economics, Anderson et al., 14th edition

#### Lectures

* Jupyter Notebooks
* Python applications

#### Exercises

* In-class exercises
* Problem sets and assignments

---

### Rules

* Be prepared for class.
* Complete assigned readings and review previous lectures.
* Attendance and active participation are important for achieving the learning goals.
* Interactive classes and cold calls should be expected.
* Explain the reasoning behind calculations and software output.
* Follow the University policies on attendance, academic integrity, and student conduct.
* AI may be used for permitted coursework when its use is disclosed and understood.
* AI is prohibited in examinations and quizzes.

---

### Grading

| Component | Weight |
|---|---:|
| Midterm examination | 20% |
| Final examination | 40% |
| Group project/assignment | 10% |
| Two quizzes | 20% |
| Class participation | 10% |

---

# Questions?

---

## Lecture 01

---

### What Is Statistics?

*   Numerical facts that summarize a situation, such as average waiting time, market share, median salary, or a customer-retention rate.

*   The art and science of collecting, analyzing, presenting, and interpreting data to support understanding and decisions.



Business question $\quad \Rightarrow \quad$ Data $\quad \Rightarrow \quad$ Analysis $\quad \Rightarrow \quad$ Interpretation $\quad \Rightarrow \quad$ Decision

A precise number can still be uninformative or misleading when the question, measurement, or data-collection process is poorly defined.

---

### Applications in Business and Economics

| Area | Example question | Possible data |
|---|---|---|
| Accounting | Which transactions require additional audit attention? | Invoice values, approval paths, exceptions |
| Economics | How might inflation affect household spending? | Prices, income, consumption over time |
| Finance | Is the expected return sufficient for the risk? | Prices, returns, firm characteristics |
| Marketing | Which campaign generates more conversions? | Exposure, clicks, purchases, customer segment |
| Operations | When and where do service delays occur? | Arrival time, waiting time, staffing |
| Human resources | What factors are associated with employee turnover? | Tenure, role, compensation, exit status |

**The same methods appear in different areas because the underlying questions concern comparison, association, prediction, and uncertainty.**

---

### Exercise 1

A logistics company introduced routing software in January. The average reported delivery time was lower from January through March than during the previous three months. Management attributes the entire improvement to the software.

1. What should one row of the analysis represent?

2. What information is needed in addition to delivery time?

3. Which comparisons would make the evidence more informative?

4. What alternative explanations should be considered?

5. What evidence would support a stronger conclusion about the effect of the software?

---

### Data and Data Sets

* Data: facts and figures collected for analysis and interpretation.

* Data set: all data collected for a particular study.

* Element: the entity on which data are collected.

* Variable: a characteristic recorded for each element.

* Value: the recorded measurement or label for one variable.

* Observation: the complete set of recorded values for one element, usually one row of a rectangular data table.

For a complete rectangular data set with n observations and p variables, the table contains n × p recorded values, excluding headers.

---

### Data and Data Sets: AUI Campus Café

Suppose the café records a small sample of lunch-period transactions. Each row represents one transaction.

| Variable | Meaning |
|---|---|
| transaction_id | Transaction identifier |
| channel | In-store or mobile order |
| category | Main product category |
| wait_minutes | Minutes from order to collection |
| amount_mad | Transaction amount in Moroccan dirhams |
| satisfaction | Customer rating from 1 to 5 |
| returning_customer | Whether the customer had purchased previously |

This is an invented classroom data set. It is not evidence about actual AUI customers.

---

In [1]:
import pandas as pd
from IPython.display import display

cafe = pd.DataFrame({
    "transaction_id": [f"T{i:02d}" for i in range(1, 13)],
    "channel": ["In-store", "Mobile", "In-store", "In-store", "Mobile", "In-store", "Mobile", "In-store", "In-store", "Mobile", "In-store", "Mobile"],
    "category": ["Coffee", "Meal", "Snack", "Meal", "Coffee", "Meal", "Snack", "Coffee", "Meal", "Meal", "Snack", "Coffee"],
    "wait_minutes": [7.5, 4.0, 6.5, 12.0, 3.5, 10.0, 5.0, 8.5, 14.0, 6.0, 9.5, 4.5],
    "amount_mad": [28, 62, 21, 74, 31, 69, 24, 35, 81, 66, 22, 33],
    "satisfaction": [4, 5, 4, 2, 5, 3, 4, 3, 2, 4, 3, 5],
    "returning_customer": [True, True, False, True, True, False, True, True, False, True, False, True]
})

display(cafe)

,transaction_id,channel,category,wait_minutes,amount_mad,satisfaction,returning_customer
0,T01,In-store,Coffee,7.5,28,4,True
1,T02,Mobile,Meal,4.0,62,5,True
2,T03,In-store,Snack,6.5,21,4,False
3,T04,In-store,Meal,12.0,74,2,True
4,T05,Mobile,Coffee,3.5,31,5,True
5,T06,In-store,Meal,10.0,69,3,False
6,T07,Mobile,Snack,5.0,24,4,True
7,T08,In-store,Coffee,8.5,35,3,True
8,T09,In-store,Meal,14.0,81,2,False
9,T10,Mobile,Meal,6.0,66,4,True


### Elements, Variables, Values, and Observations

In the café data set:

* The element is a lunch-period transaction.

* The table contains 12 observations.

* Each row contains the recorded values for one transaction.

* The variable wait_minutes has the value 12.0 for transaction T04.

* The identifier distinguishes transactions but is not a measured quantity.

The element is determined by the question. 
    
- A product-level study and a transaction-level study may require different tables built from the same operational records.

---

### Exercise 2

The café exports sales data with one row for every product purchased. 

-   A transaction containing three products therefore appears on three rows. 

-   Transaction waiting time and satisfaction are repeated on all three rows.

**Questions**

1. If the question concerns waiting time, what should the element and observation be?

2. What happens if the analyst calculates the mean waiting time directly from the product rows?

3. Would the product rows be suitable for studying the popularity of individual products?

4. Is transaction_id a quantitative variable merely because its values contain numbers?

5. How should the exported data be reorganized for the waiting-time study?

---

### Data Organization and Software Types

A useful rectangular data set normally follows two principles:

* Each row represents one kind of element.

* Each column represents one variable.

Spreadsheet titles, totals, comments, and multiple kinds of elements should not be mixed into the observation rows.

*Software storage types such as text, integer, decimal, and Boolean describe how values are stored.*

-   Statistical meaning depends on what the variable represents. 

-   A numeric code can still be categorical, and an ordered rating can still require cautious interpretation.

---

In [2]:
inspection = pd.DataFrame({
    "variable": cafe.columns,
    "software_type": [str(dtype) for dtype in cafe.dtypes],
    "missing_values": cafe.isna().sum().values,
    "distinct_values": cafe.nunique().values
})

display(inspection)

,variable,software_type,missing_values,distinct_values
0,transaction_id,object,0,12
1,channel,object,0,2
2,category,object,0,3
3,wait_minutes,float64,0,12
4,amount_mad,int64,0,12
5,satisfaction,int64,0,4
6,returning_customer,bool,0,2


### Categorical and Quantitative Data

Categorical data identify a quality or group. Their values may be names or numeric codes.

* channel, category, and returning_customer are categorical.

* transaction_id is an identifier and is usually treated as categorical.

Quantitative data indicate how much or how many. Arithmetic differences have a meaningful interpretation.

* wait_minutes and amount_mad are quantitative.

* Quantitative data may be discrete or continuous.

    * Discrete data count how many, such as the number of products in a transaction.

    * Continuous data measure how much, such as wait_minutes.

Satisfaction uses numbers, but those numbers primarily indicate ordered categories. Numeric appearance alone does not determine statistical meaning.

---

### Scales of Measurement

| Scale | Information carried by the values | Meaningful comparisons |
|---|---|---|
| Nominal | Category identity | Same or different |
| Ordinal | Category identity and order | Higher or lower |
| Interval | Order and meaningful differences | Differences |
| Ratio | Order, differences, and a meaningful zero | Differences and ratios |

The scale of measurement affects which summaries, graphs, and statistical methods are appropriate.

---

### Nominal and Ordinal Scales

#### Nominal Scale

* Values distinguish categories without ranking them.

* Examples: ordering channel, product category, nationality, and account type.

* A numeric category code does not create quantitative data.

#### Ordinal Scale

* Values distinguish and rank categories.

* Examples: credit rating, class standing, and satisfaction category.

* The distance between adjacent categories is not necessarily constant.

---

### Interval and Ratio Scales

#### Interval Scale

* Differences between values are meaningful.

* Zero is a reference point rather than the absence of the measured quantity.

* Examples: temperature in Celsius and calendar year.

#### Ratio Scale

* Differences and ratios are meaningful.

* Zero indicates the absence of the measured quantity.

* Examples: waiting time, transaction amount, revenue, age, and units sold.

*A waiting time of 20 minutes is twice as long as 10 minutes. A temperature of 20°C is not twice as hot as 10°C because 0°C does not represent the absence of temperature.*

---

### Cross-Sectional and Time-Series Data

Cross-sectional data describe different elements at the same or approximately the same point in time.

* Revenue for 40 branches during August 2026

* Credit ratings for 60 companies on the same date

Time-series data describe the same variable over several time periods.

* Monthly revenue for one branch from 2024 through 2026

* Daily exchange rates during one year

When the same branches are observed repeatedly over time, the complete structure is panel data, also called longitudinal data. 

- Panel data combine cross-sectional and time-series dimensions.

---

### Exercise 3

A bank follows 40 branches monthly for two years. The table contains one row per branch-month and includes branch_id, region_code, internal_credit_rating, opening_year, monthly_revenue_mad, complaint_count, and manager_satisfaction from 1 to 5.

1. Identify the element represented by one row. Why is branch alone not a complete answer?

2. Classify the variables by statistical type and measurement scale.

3. Explain why opening_year supports meaningful differences but not meaningful ratios.

4. Name the structure of the complete table. How could it be used to construct a cross-sectional data set and a time-series data set?

5. What information would be lost if the table were reduced to one row per branch using two-year averages?

---

### Exercise 3: Data Excerpt

The table shows three branches observed over three months from the larger data set.

| branch_id | month | region_code | internal_credit_rating | opening_year | monthly_revenue_mad | complaint_count | manager_satisfaction |
|---|---|---:|---|---:|---:|---:|---:|
| B01 | 2026-01 | 10 | A | 2018 | 860,000 | 12 | 3 |
| B01 | 2026-02 | 10 | A | 2018 | 905,000 | 8 | 4 |
| B01 | 2026-03 | 10 | A | 2018 | 890,000 | 10 | 4 |
| B02 | 2026-01 | 20 | BBB | 2012 | 1,120,000 | 6 | 4 |
| B02 | 2026-02 | 20 | BBB | 2012 | 1,075,000 | 9 | 3 |
| B02 | 2026-03 | 20 | BBB | 2012 | 1,160,000 | 5 | 4 |
| B03 | 2026-01 | 30 | BB | 2021 | 640,000 | 15 | 2 |
| B03 | 2026-02 | 30 | BB | 2021 | 720,000 | 11 | 3 |
| B03 | 2026-03 | 30 | BB | 2021 | 695,000 | 13 | 3 |

---

### Data Sources

#### Existing Sources

* Internal records: sales, accounting, inventory, employees, customers, and operations

* External sources: government agencies, industry associations, commercial databases, and research publications

#### New Data Collection

* Surveys and interviews

* Direct observation

* Experiments

Availability does not establish suitability. Data collected for another purpose may use a different population, definition, frequency, or unit of analysis.

---

### Observational and Experimental Studies

In an observational study, the analyst observes variables without assigning the conditions being compared.

* Customers choose mobile or in-store ordering.

* Firms choose whether and when to adopt a technology.

In an experimental study, the researcher assigns one or more conditions and observes the resulting outcomes.

* Comparable customers are randomly assigned different website designs.

* Similar production periods are assigned different operating procedures.

Observational data can reveal important associations. A credible causal conclusion requires stronger design and careful treatment of alternative explanations.

---

### Data-Acquisition Considerations

* Relevance: Were the data collected for the current question?

* Definition: Are variables measured consistently across elements and periods?

* Coverage: Which elements, locations, or periods are absent?

* Accuracy: Can recording, coding, or processing errors be detected?

* Timeliness: Are the data still useful for the decision?

* Cost: Is additional information worth the time and expense?

* Access and responsibility: Is the collection and use of the data permitted and appropriate?

**The quality of an analysis cannot exceed the quality and relevance of the data used.**

---

### Descriptive Statistics

Descriptive statistics summarize the data that were observed.

Common forms include:

* Tabular summaries

* Graphical summaries

* Numerical summaries

Examples from the café data include the number of recorded transactions, the distribution of product categories, and the mean waiting time in the observed rows.

A descriptive result may be accurate for the recorded data while still being unrepresentative of a broader population.

---

### Population, Sample, Census, and Sample Survey

* Population: the complete set of elements relevant to a study.

* Sample: a subset of the population from which data are collected.

* Census: data collection from every element in the population.

* Sample survey: data collection from a sample of the population.

If the café manager wants to understand all Fall 2026 lunch transactions, those transactions form the population. The 12 classroom rows serve as a hypothetical sample for illustration; they were not collected from that population.

A population is defined by the question, including the relevant elements, place, and time period. It is not defined merely by the file currently available.

---

### Statistical Inference

Statistical inference uses sample evidence to learn about a population.

* A parameter describes a population.

* A statistic is calculated from a sample.

* Estimation uses a sample statistic to estimate a population parameter.

* Hypothesis testing evaluates a claim about a population parameter.

Inference requires more than a large number of observations. The sample must connect credibly to the population, and uncertainty must be assessed.

---

In [3]:
sample_a = cafe.sample(n=5, random_state=2301)
sample_b = cafe.sample(n=5, random_state=2302)

comparison = pd.DataFrame({
    "data": ["All 12 rows", "Sample A", "Sample B"],
    "observations": [len(cafe), len(sample_a), len(sample_b)],
    "mean_wait_minutes": [cafe["wait_minutes"].mean(), sample_a["wait_minutes"].mean(), sample_b["wait_minutes"].mean()],
    "mobile_share": [cafe["channel"].eq("Mobile").mean(), sample_a["channel"].eq("Mobile").mean(), sample_b["channel"].eq("Mobile").mean()]
})

display(comparison.round(3))
display(cafe.groupby("channel")["wait_minutes"].agg(["count", "mean"]).round(2))

,data,observations,mean_wait_minutes,mobile_share
0,All 12 rows,12,7.583,0.417
1,Sample A,5,7.800,0.400
2,Sample B,5,8.700,0.200


,count,mean
channel,,
In-store,7,9.71
Mobile,5,4.60


### Exercise 4

In the hypothetical 12-row café data set, the mean waiting time is approximately 9.7 minutes for in-store orders and 4.6 minutes for mobile orders. A manager concludes that mobile ordering reduces waiting time by approximately 5.1 minutes for all AUI lunch transactions.

1. Separate the descriptive part of the statement from the inferential part.

2. Define the population implied by the manager's statement.

3. What do the different results from Sample A and Sample B demonstrate?

4. Why does the difference between ordering channels not establish a causal effect?

5. What additional data or design changes would make the comparison more credible?

---

### Analytics

Analytics transforms data into insight for decision making.

* Descriptive analytics: What happened?

* Predictive analytics: What is likely to happen, or how is one variable associated with another?

* Prescriptive analytics: Which action should be taken under stated objectives and constraints?

For café operations:

* Summarize waiting times by period.

* Predict congestion from order mix and staffing.

* Choose a staffing plan subject to cost and service targets.

The categories are connected, but a descriptive pattern alone does not determine the best action.

---

### Big Data, Data Mining, and Model Reliability

Big data are commonly described by:

* Volume: the amount of data

* Velocity: the speed of collection and processing

* Variety: the range of structured and unstructured data

Data mining uses statistical and computational methods to discover useful patterns and develop predictions from large data sets.

A model that fits the data used to build it may perform poorly on new data. 
-   Separating training data from test data helps evaluate whether a pattern is repeatable. 

-   Large data sets do not eliminate selection bias, measurement problems, overfitting, or misleading causal interpretations.

---

### Computers and Statistical Analysis

Computers make it possible to store, process, visualize, and analyze data efficiently.

This course will use:

* Excel for spreadsheets and familiar business workflows

* Python for reproducible data manipulation and analysis

* Jupyter Notebooks for combining explanations, calculations, tables, and figures

Software can execute a procedure correctly while the analysis remains conceptually wrong. 
-   The analyst is responsible for the question, data, assumptions, method, interpretation, and communication.

---

### Exercise 5

Ethical statistical practice requires fair, thorough, objective, and transparent work in data collection, analysis, presentation, and interpretation.

A lender develops 25 rules for identifying customers likely to miss a payment. It reports only the rule with the highest accuracy on the historical data used to create the rules. The lender also removes several missed payments after describing them as unusual, but does not document the exclusions.

1. Which parts of the analysis may produce an overly favorable result?

2. Why is high accuracy on the development data insufficient?

3. What information should be disclosed with the reported result?

4. How should the lender evaluate whether the rule is reliable?

5. What responsibilities remain even if every calculation is technically correct?

---

### What is next?

* Descriptive Statistics I
    * Tabular and Graphical Displays
    * Reading(s):
        * SBE: Ch. 2

---

### Exercise 6

Atlas Retail operates 40 stores. Six store managers volunteer to test express checkout. The company records 1,240 transactions during weekday lunch periods. Its exported table has one row per product purchased, so transaction waiting time is repeated on every product row. Using the exported rows, an analyst reports mean waiting times of 3.8 minutes for express checkout and 6.1 minutes for regular checkout.

Atlas Retail's chief operating officer asks whether express checkout should be introduced across all stores. Audit the analyst's evidence and prepare a recommendation. State what can be concluded from the available data, what must be corrected, and what additional evidence management should require before acting.

---

### Exercise 7

Sahara Streaming wants to predict whether a subscriber will cancel next month. Its file has one row per program watched, so subscriber characteristics and monthly spending are repeated. Variables include subscriber_id, membership_plan, satisfaction from 1 to 5, signup_year, monthly_spending_mad, hours_watched, and cancellation_next_month. Analysts build many prediction rules and report 96% accuracy using the same data on which the selected rule was developed. In the data, 96% of subscribers did not cancel.

Management is ready to deploy the prediction rule. Prepare a technical recommendation on whether the reported evidence supports deployment. If it does not, explain how the analysis should be rebuilt and what evidence would be required to justify using the model.

---

### Exercise 8

A hotel group introduces an energy-management program at one coastal hotel in June. It compares that hotel's electricity use from June through August with its use from March through May. Electricity consumption falls by 12%, and management announces that the program will reduce electricity use by 12% across all 18 hotels. The coastal hotel had unusually mild summer weather, occupancy fell, and several floors were renovated during the study period.

The executive committee asks whether the program should be introduced across all 18 hotels. Prepare an analytical recommendation based on the available evidence.

---